
# mode: Dbg, Rel 
mode : Dbg

# 1 or 2, 1 go left, 2 go right,, 0 stay forever
strategy: 0

# gimbal_action_mode:
# 0: not test gimbal, make decision only
# 1: test_fire mode, use rqt to control gimbal, if fire_times == -1, then can control rel_yaw,rel_pitch, or can control fire_times

gimbal_action_mode: 0

if_ignore_elesys: False
if_ignore_brother: True
if_auto_recognize_friend: True  # Must be True in 3v3, False in 1v1, and must add 7 to decision maker params if True
if_main_head: False
if_show_img_local: False
if_show_img_remote: False
# enemy armor color: blue or red
armor_color : blue



if_yolov5: True
# WARNING : armor_name depends on if_yolov5 !!!

enemy_car_list : 
- armor_name: "00"
  armor_nums: 2
- armor_name: "01"
  armor_nums: 2
- armor_name: "03"
  armor_nums: 2
- armor_name: "04"
  armor_nums: 2
- armor_name: "10"
  armor_nums: 2
- armor_name: "11"
  armor_nums: 2
- armor_name: "13"
  armor_nums: 2
- armor_name: "14"
  armor_nums: 2

# armor_distance : the distance between 2 opposite armors, unit: m
# if armor_nums == 4: [x_distance, y_distance], x distance means maximum efficency direction of robot chassis 
# if armor_nums == 2: [no_meaning, distance] 



# Warning :
# global logger level need to be set in os_op/global_logger.py


# Notice :
# rviz2 yaw = 4.72318 pitch = 0.2 , in this sight, x if right, y is in, z is up


# Notice :
# config ballistic config : gun_pivot_height !!!

camera_type: "mv"   # "mv" or "gg"
flip_img_code: -1 # -1 for mv, 1 for gg
img_show_wid: 640
img_show_hei: 384
node_webcam_ms: 10






